.travis.yml

```diff
@@ -288,6 +288,10 @@ jobs:
           upload_dir: "enterprise/RM/${RELEASE_VERSION}"
           on:
             all_branches: true
+        - provider: script
+            script: bash scripts/travis/copy_share_image_to_dockerHub.sh
```

Comment:
please use either snakecase or camelcase, not a mix of both > docker_hub

Reply:
Updated File Name